apps/central-scan/backend/src/central_scanner_app.ts

```diff
@@ -126,10 +122,17 @@ function buildApi({
 
     /* istanbul ignore next - only used by Cypress */
     async configureWithSampleBallotPackageForIntegrationTest(): Promise<void> {
-      const fileContents = await fs.readFile(
-        '../integration-testing/cypress/fixtures/ballot-package.zip'
+      if (process.env.CI !== 'true') {
+        throw new Error(
+          'This function is only intended to be used in CI environments'
```

Comment:
Why?

Reply:
I guess we do run it sometimes locally. Mostly I was trying to disallow this in production.